compute_period : 50
sensor_threshold: 20.0

rotate_right: [0.2, 0.2, 0.2]
rotate_left: [-0.2, -0.2, -0.2]
move_forward: [0.2, 0.2, 0.0]